src/business-logic/services/interfaces/rundown-service.ts

```diff
@@ -6,6 +6,7 @@ import { InTransition } from '../../../model/value-objects/in-transition'
 export interface RundownService {
   deleteRundown(rundownId: string): Promise<void>
   activateRundown(rundownId: string): Promise<void>
+  enterRehearsalOnRundown(rundownId: string): Promise<void>
```

Comment:
I decided to add `Rundown` to the name since that's the convention all the other methods are using. (Atleast for delete, activate and deactivate).
I agree the name is a bit weird. I have changed it to `enterRehearsal`.